README.md

```diff
@@ -120,5 +120,22 @@ Test:
 $ docker run todo-app:test
 ```
 
+## Deploying the app
+### Deploying a new image to Docker Hub
+Docker image can be found [here](https://hub.docker.com/r/clahur/todo-app/tags).
+
+1. Log into DockerHub locally, with ```docker login```
+2. Build the image, with 
+    ```bash
+    $ docker build --target production --tag <user_name>/todo-app:latest .
+    ```
+3. Push the image, with 
+    ```bash
+    $ docker push <user_name>/todo-app:latest
+    ```
+
+### Updating the container
```

Comment:
Minor, but it would be nice to quickly list the websites URL - it might sound stupid but I've definitely joined projects and then struggled to work out where they actually run in the wild before!
<https://clahurtodoapp.azurewebsites.net/>